src/components/ChangelogDialog/components/Item/Item.tsx

```diff
@@ -32,10 +46,28 @@ export function Item({className, data, onStoryClick, onLinkClick}: ItemProps) {
         }
     }, [data.storyId, onStoryClick]);
 
+    const formattedDate = React.useMemo(() => {
+        let value;
+        if (!data.date) return value;
+
+        const locale = settings.getLocale();
```

Comment:
How user is supposed to set it? Maybe we use current language as a locale?

Reply:
and @gravity-ui/date-utils may be used in parent scope, so, we don't change settings

Reply:
It's a weak bound. It can have this package in parent scope, but sometimes it cant have.

Reply:
Add uikit language